datasets/switchboard.py

```diff
@@ -0,0 +1,293 @@
+"""
+Switchboard is conversational telephony speech with 8 Khz audio files. The training data consists of
```

Comment:
maybe give the link
https://catalog.ldc.upenn.edu/LDC97S62
in the description to make sure it is clear which version of SWB is used. Or at least give the LDC catalogue number.